examples/with-jest/example.test.ts

```diff
@@ -1,7 +1,3 @@
-/**
- * @jest-environment node
- */
-
 it('receives a mocked response to a REST API request', async () => {
```

Comment:
The `with-jest` example is missing a README.

Reply:
I've kept the original README

Reply:
Once again, GitHub shenanigans.